docs/source/relayer.rst

```diff
@@ -1,2 +1,273 @@
 Relayer
 =======
+
```

Comment:
Start with summary and table of content:
To setup your own relayer ...
There are to ways: [form cmd]() and [programmatically].

If you would like to have your own domain jump to the section on setting up [Custom ENS Domain].
If you would like to build your own rly go to [Building Custom Relayer] setion.